src/runmodes.c

```diff
@@ -252,47 +252,64 @@ void RunModeRegisterRunModes(void)
 }
 
 /**
- * \brief Lists all registered runmodes.
+ * \brief Lists all registered runmodes (optimized table output).
  */
 void RunModeListRunmodes(void)
 {
-    printf("------------------------------------- Runmodes -------------------"
-           "-----------------------\n");
-
-    printf("| %-17s | %-17s | %-10s \n",
-           "RunMode Type", "Custom Mode ", "Description");
-    printf("|-----------------------------------------------------------------"
-           "-----------------------\n");
-    int i = RUNMODE_UNKNOWN + 1;
```

Comment:
As ticket #5711 is for adding a hint for the lack of a runmode, please remove these changes.
Another feedback related to this: unrelated changes belong in different commits. ;)